raw-spans-grouper/helm/values.yaml

```diff
@@ -112,6 +112,9 @@ rawSpansGrouperConfig:
         bootstrapServers: "bootstrap:9092"
         schemaRegistryUrl: "http://schema-registry-service:8081"
         cacheMaxBytesBuffering: 16777216 # 16MB
+        commitIntervalMs: 5000 #default
+        producerBatchSize: 16384 #default
```

Comment:
From the [documentation](http://cloudurable.com/blog/kafka-tutorial-kafka-producer-advanced-java-examples/index.html#:~:text=Kafka%20Producer%3A%20Batching%20by%20Size,sent%20to%20the%20same%20partition.), it looks like 16384=16KB is the default and we are using the conditional so I wonder why we are specifying this as default. I'd rather not.